pl_engine/pl_server/src/main/antlr/PlcParser.g4

```diff
@@ -40,7 +40,7 @@ create_routine
 
 routine_definition
     : (PROCEDURE | FUNCTION) identifier ( (LPAREN parameter_list RPAREN)? | LPAREN RPAREN ) (RETURN type_spec)?
-      (IS | AS) (LANGUAGE PLCSQL)? seq_of_declare_specs? body (SEMICOLON)?
+      (authid_spec)? (IS | AS) (LANGUAGE PLCSQL)? seq_of_declare_specs? body (SEMICOLON)?
```

Comment:
Because the routine_definition non-terminal is shared by local procedure/functions, we need to check if the authid_spec is present in a local procedure/function definition and report an error if present.
Note that a similar check is done for LANGUAGE PLCSQL clause in visitRoutine_definition() in ParseTreeConverter.java.